config/orm.js

```diff
@@ -0,0 +1,75 @@
+const db = require('./connection.js');
+const userData = mongoose.model("userData", userSchema);
+
+module.exports = {
+
+    createUser: function (req, res) {
+        db.userData
+        .create({username: x, password: y, email: z })
+        .then(userData => res.send(userData))
+        .catch(err => console.log(err));
+    },
+    loginUser: function (req, res) {
+        db.userData
+        .findOne ({username: x}, {password: req.body.data})
+        .then (data => res.send(data))
+        .catch(err => res.status(422).json(err));
+    },
+
+    getScoreTA: function (req, res) {
+        db.userData
+        .findOne({username: x}, {highscore_TA})
```

Comment:
assuming this is to get all the highscore from all the users in an descending order
```
.findOne({username: req,body.data.username})
.sort({highscore_TA: -1})
```